docs/sources/next/using-k6/javascript-compatibility-mode.md

```diff
@@ -1,44 +1,71 @@
 ---
-title: JavaScript Compatibility Mode
+title: JavaScript compatibility mode
+menuTitle: JavaScript mode
 excerpt: 'k6 supports running test scripts with different ECMAScript compatibility modes using --compatibility-mode'
 _build:
   list: false
```

Comment:
This page is currently hidden from the navigation. Do we want to update this so users can find it more easily?

Reply:
No. We still want to be hidden as we don't recommend using this option. This page is referenced from other related pages.